packages/bot-skeleton/src/services/tradeEngine/trade/index.js

```diff
@@ -22,7 +21,7 @@ const watchBefore = store =>
         store,
         stopScope: constants.DURING_PURCHASE,
         passScope: constants.BEFORE_PURCHASE,
-        passFlag: 'proposalsReady',
+        passFlag: '',
```

Comment:
We might want to name the `passFlag` instead of keeping it empty.

Reply:
@prince-deriv If we name this variable, we want to track it, but right now we don't want to track changes of any flag before the block purchase conditions, but it's nice to be able to know **the stage**, and **the** **phase** of the **bot**. So I decided to leave the flag without a name => don't track the flag.
If you have any other suggestions, I'd be happy to consider them 🙂